src/pydvl/value/shapley/montecarlo.py

```diff
@@ -40,23 +41,21 @@
 import numpy as np
 from numpy.typing import NDArray
 
-from pydvl.utils import (
-    MapReduceJob,
-    ParallelConfig,
-    Utility,
-    get_running_avg_variance,
-    init_parallel_backend,
-    maybe_progress,
-    random_powerset,
-)
-from pydvl.value import ValuationResult, ValuationStatus
-
+from ...utils.config import ParallelConfig
```

Comment:
Seeing this makes me realize that absolute imports are more readable.

Reply:
Agreed, I think pycharm suggested it and I kept using them, thinking "maybe we'd want to rearrange stuff later on", but I'll revert to absolute ones in the 2nd PR